infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesDeployments.java

```diff
@@ -779,7 +780,7 @@ private Pod awaitAndGetPod(Deployment deploy) throws InfrastructureException {
                         String.format("Failed to get ReplicaSet controlling Pod %s", podName)));
 
     ReplicaSet replicaSet =
-        clientFactory.create(workspaceId).extensions().replicaSets().withName(replicaSetName).get();
+        clientFactory.create(workspaceId).extensions().replicaSets().inNamespace(namespace).withName(replicaSetName).get();
```

Comment:
This line looks too long. Please make sure that it is well formatted (`mvn fmt:format`)